examples/threadcall_detect_in_image.jl

```diff
@@ -0,0 +1,56 @@
+using Images
+#Additional required packages to run this example.
+#Pkg.add("Images")
+
+using AprilTags
+
+# Simple method to show the image with the tags
+function showImage(image, tags)
+    # Convert image to RGB
+    imageCol = RGB.(image)
+    #traw color box on tag corners
+    foreach(tag->drawTagBox!(imageCol, tag), tags)
+    imageCol
+end
+
+
+##
+# Create default detector
+detector = AprilTagDetector()
+
+#Run against a file
+image = load(dirname(Base.source_path()) *"/../data/tagtest.jpg")
+
+@sync begin
```

Comment:
Yes, I think this is the right way to do it.  Haven't worked with `@threadcall` before but understand from docs that it is non-blocking.